src/features/books/booksReducer.js

```diff
@@ -0,0 +1,40 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = {
+  id: 1,
+  books: [{ id: 0, title: 'book1', author: 'Willam' }, { id: 1, title: 'book2', author: 'Omar' }],
+};
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
```

Comment:
- [ ] Also this file should be in the redux folder.

![redux-book](https://user-images.githubusercontent.com/61562626/149918777-a26c0966-8894-4edd-8307-3f754617299d.jpeg)